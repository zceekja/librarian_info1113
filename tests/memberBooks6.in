add book book1.csv 111111
add book book1.csv 111112
add member Tony Stark
rent 100000 111111
rent 100000 111112
relinquish 100000 111111
member books 100000