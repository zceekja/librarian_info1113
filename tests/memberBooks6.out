user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Successfully added: The Hobbit (J.R.R. Tolkien).

user: Success.

user: Success.

user: Success.

user: Success.

user: The Hobbit (J.R.R. Tolkien)

user: 